rasa/core/policies/ted_policy.py

```diff
@@ -644,11 +642,7 @@ def predict_action_probabilities(
             tracker, domain, interpreter
         )
         model_data = self._create_model_data(tracker_state_features)
-
-        data_generator = RasaBatchDataGenerator(
-            model_data, batch_size=len(tracker_state_features)
-        )
-        output = self.model.predict(data_generator)
```

Comment:
is `self.model.predict` ever still used?

Reply:
it can be used, we don't use it, but you would in your custom code if you want to connect some callbacks